### PR TITLE
Add support for --ssl and --authenticationDatabase options

### PR DESCRIPTION
**Pull request by cavvia**
This adds `--ssl` and `--authenticationDatabase` support to all commands.

### CODE REVIEW

File: lib/mongoid/shell/properties/authentication_database.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module Mongoid
+  module Shell
+    module Properties
+      module AuthenticationDatabase
```
Comment (dblock) [inline lib/mongoid/shell/properties/authentication_database.rb]: Why isn't this called `AuthSource`?

**Comment by dblock** [comment]
This looks good. See my comment on class na.e. You also need to add it to README I believe, please.

File: lib/mongoid/shell/properties/authentication_database.rb (hunk 1 above)
Comment (joeyAghion) [reply lib/mongoid/shell/properties/authentication_database.rb]: The naming across the `mongo` driver (`auth_source`) and command line utilities (`--authenticationDatabase`) is inconsistent. This tries to stay consistent with the command line naming, but we can update if we misinterpreted that convention.

Comment (dblock) [reply lib/mongoid/shell/properties/authentication_database.rb]: I see, we can leave this as is.

**Comment by joeyAghion** [comment]
README updated!

**Comment by cavvia** [comment]
Do let us know when a new point release is out so we can stop pointing to our fork.

**Pull request merged**